### ISSUE TITLE
Sorting inside @api blocks

### ISSUE THREAD

**Issue by sebastiaanluca** [apidoc/apidoc#228]
How exactly is the result of every `@api` block generated? Can it be customized? Because now I have something like the following, but it doesn't render the HTML in the same order as I wrote it:

``` php
    /**
     * @api {get} /users Description here
     * @apiName getUsers
     * @apiGroup Users
     *
     * @apiUse Block1
     * @apiUse Block2
     *
     * @apiSuccess (Success200) {integer} id Description
     * @apiSuccess (Success200) {string} name Description
     *
     * @apiSuccessExample
    */
```

The result:
- Block 2
- Both @apiSuccess'
- Block 1
- @apiSuccessExample

**Comment by rottmann** [comment]
The exactly order is not considered at the moment. It uses the file order. Block2 is defined before Block1.
I add this todo.

**Comment by Quagh** [comment]
What do you mean by file order, as it's just one file, one block of apiDoc comments? The blocks are defined in the order I want them to appear, same as the `@apiUse`s.

Can you refer me to the correct file so I can implement this myself if need be?

**Comment by mauler** [comment]
@sebastiaanluca  this should solve your problem

https://github.com/apidoc/apidoc/pull/457

**Comment by NicolasCARPi** [comment]
Closing as fixed by #457
